src/webgpu/shader/execution/expression/call/builtin/round.spec.ts

```diff
@@ -43,15 +48,30 @@ export const d = makeCaseCache('round', {
       FP.f16.roundInterval
     );
   },
+  abstract: () => {
+    return FP.abstract.generateScalarToIntervalCases(
+      [
+        reinterpretU64AsF64(0x8000_0000_0000_0000n), // https://github.com/gpuweb/cts/issues/2766
```

Comment:
This is a reinterpret cast, so this generates  -0.0 in f64.

I don't think that's what you mean.

Reply:
The point of 2766 was to test integers that are larger in magnitude than can fit inside the mantissa bits.  So for f64 that would be something in excess of 2** 53 ish.

Reply:
Correct, thanks for catching this.

Reply:
arg... I updated the unittests, but not the CTS test itself :face_exhaling:

Reply:
Updated